samples/ts/21.react-media-template/src/components/InkCanvas.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC, MutableRefObject, useEffect } from "react";
+import useResizeObserver from "use-resize-observer";
+import { getInkCanvasStyles } from "../styles/styles";
+import { useVisibleVideoSize } from "../utils/useVisibleVideoSize";
+import { useEventListener } from "../utils/useEventListener";
+import { InkingManager } from "@microsoft/live-share-canvas";
+
+const REFERENCE_HEIGHT = 1080;
+
+export const InkCanvas: FC<{
+    isEnabled: boolean;
+    inkingManager: InkingManager;
+    canvasRef: MutableRefObject<HTMLDivElement | undefined>;
+}> = ({ isEnabled, inkingManager, canvasRef }) => {
+    const { ref: resizeRef, width = 1, height = 1 } = useResizeObserver();
+    const videoSize = useVisibleVideoSize(width, height);
+
+    const onMouseEvent = (event: Event) => {
+        if (isEnabled) {
+            event.preventDefault();
+        }
+    };
+
+    useEffect(() => {
+        if (videoSize && canvasRef.current) {
+            canvasRef.current.style.width = `${videoSize.width}px`;
+            canvasRef.current.style.height = `${videoSize.height}px`;
+            if (inkingManager) {
+                // Update the scale of inkingManager so that the annotations appear
+                // in the same positions for all users
+                const scale = videoSize.height / REFERENCE_HEIGHT;
+                inkingManager.scale = scale;
+            }
+        }
+    }, [videoSize, inkingManager, canvasRef]);
+
+    useEventListener("mousedown", onMouseEvent, canvasRef.current);
+    useEventListener("mouseup", onMouseEvent, canvasRef.current);
+    useEventListener("mousemove", onMouseEvent, canvasRef.current);
+    const inkStyles = getInkCanvasStyles();
+
+    return (
+        <>
+            <div className={inkStyles.root} ref={resizeRef} />
```

Comment:
This doesn't need to be a wrapper around the div below?
https://domoreexp.visualstudio.com/Teamspace/_search?action=contents&text=resizeref&type=code&lp=code-Project&filters=ProjectFilters%7BTeamspace%7DRepositoryFilters%7Bteams-graphservice%7D&pageSize=25&result=DefaultCollection/Teamspace/teams-graphservice/GBdevelop//Source/TeamsCoWatch/YouTube/src/components/player/VideoPlayer.tsx

Reply:
I think this is intentional? inking seems to work as intended